shell/packages/sandstorm-db/db.js

```diff
@@ -498,6 +498,9 @@ Notifications = new Mongo.Collection("notifications");
 //   text:         The JSON-ified LocalizedText to display in the notification.
 //   isUnread:     Boolean indicating if this notification is unread.
 //   timestamp:    Date when this notification was last updated
+//   initiatingIdentity: Identity ID of the user who initiated this notification.
```

Comment:
Perhaps these belong grouped under a `grain` object.  Not-blocking, but would be nice to know the logical invariants that should apply (e.g. "these will never exist unless `grainId` also exists")

`admin` and `appUpdates` below appear to do this sort of thing.